posthog/models/dashboard_tile.py

```diff
@@ -79,6 +79,9 @@ def save(self, *args, **kwargs) -> None:
 
         super(DashboardTile, self).save(*args, **kwargs)
 
+    def add_to_dashboard(self, dashboard: Dashboard) -> None:
```

Comment:
More like `copy_onto_dashboard`? Different identities – if I edit the new text card, the old one will still be the same.